src/elm/Auth.elm

```diff
@@ -954,6 +955,19 @@ msgToString msg =
             [ "KeyPressed" ]
 
 
+onEnter : Msg -> Html.Attribute Msg
+onEnter msg =
```

Comment:
this function is already defined in a shared module: https://github.com/cambiatus/frontend/blob/cd911ed6adb67c38624abdc2823ab3b7d60e4fd8/src%2Felm%2FUtils.elm#L58